brainscore/submission/ml_pool.py

```diff
@@ -24,10 +24,24 @@ def __contains__(self, item):
 
 
 class MLBrainPool(UniqueKeyDict):
-    def __init__(self, base_model_pool, model_layers):
+    def __init__(self, base_model_pool, model_layers, vs_model_param=None):
```

Comment:
What is `vs_model_param` for? Intuitively it seems like this parameter should not be here. If you want to add a specific new model you can implement it outside this generic `MLBrainPool`

Reply:
This is what I was most confused about.

I guess you wanted me to do the implementation such that it can be aligned with this generic MLBrainPool. I can create a different one as well. In fact, I think that will be much better. Let me know what seems suitable to you.